src/nvim/CMakeLists.txt

```diff
@@ -928,7 +928,8 @@ set(GEN_EVAL_FILES
 
 add_custom_command(
   OUTPUT ${GEN_EVAL_FILES}
-  COMMAND ${PROJECT_SOURCE_DIR}/scripts/gen_eval_files.lua
+  COMMAND nvim
```

Comment:
```suggestion
  COMMAND $<TARGET_FILE:nvim>
```

Reply:
Why is this necessary? [CMake docs](https://cmake.org/cmake/help/latest/command/add_custom_command.html) say 

>If COMMAND specifies an executable target name (created by the [add_executable()](https://cmake.org/cmake/help/latest/command/add_executable.html#command:add_executable) command), it will automatically be replaced by the location of the executable created at build time

And this seems to work as-is in my testing.

Reply:
I think this should work as is, but IIRC cmake only recognizes that `nvim` refers to the executable if it's the first argument after `COMMAND` and not otherwise, while `<TARGET_FILE:nvim>` will always refer to the executable regardless of where it's specified, so I mostly feel it's more explicit that way (and I don't have to remember cmake rules). We can keep this, mostly a nitpick/preference.